Vungle/CHANGELOG.md

```diff
@@ -1,4 +1,9 @@
 ## Changelog
+
+  *  6.5.1.0
```

Comment:
shouldn't we update our .aar as well?

Reply:
Not sure about MoPub's policy, I do see some AdNetworks adding it in their PR, if after upstream PR submission there would be additional changes during PR review the .aar could be outdated, unless we or MoPub later update it in event of any further changes.